lib/sda/overlay.go

```diff
@@ -283,7 +280,7 @@ func (o *Overlay) Close() {
 	o.nodeLock.RLock()
 	defer o.nodeLock.RUnlock()
 	for _, n := range o.nodes {
-		if err := n.ProtocolInstance().Shutdown(); err != nil {
+		if err := n.Close(); err != nil {
```

Comment:
Doesn't this introduce some problems (or do we plan to get rid of `Shutdown` again)?